### PR TITLE
M600 Filament Change documentation

### PR DESCRIPTION
**Pull request by petrzjunior**
Add requested M600 Filament change documentation. Hope I have done everything correctly, if not correct me please.
I added it to new category named `filament`, where I would suggest putting future articles like manual filament change, filament runout etc. according to future `filament` singleton.

I also changed the template for G-Code to diplay author as other pages do. It doesn't break the design and seems helpful to me.

### CODE REVIEW

**Comment by jbrazio** [comment]
Looks OK @petrzjunior, do you mind only rebasing against the latest master branch ?

Inside your local tree do the following:
```bash
$ git remote add upstream https://github.com/MarlinFirmware/MarlinDocumentation.git
$ git fetch --all --prune
$ git rebase -i upstream/master
$ git push --force
```

File: _gcode/M600.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+author: clexpert, petrzjunior
+category: [ 'gcode' ]
+
+
+title: Filament Change
+brief: Automatically change filament
+
+experimental: false
+since: 1.0.0-beta
```
Comment (jbrazio) [inline _gcode/M600.md]: Version should be 1.1.0-RC7

Comment (petrzjunior) [inline _gcode/M600.md]: Oh, is it version of the documentation? I spend half an hour searching when `M600` was first implemented.

**Comment by petrzjunior** [comment]
OK, rebased, fixed the version. Could I know why was it rebased? Not so experienced to git so don't know what's the point.

**Comment by jbrazio** [comment]
Weird, I asked for rebase because I'm seeing changes to _includes/gcode-info.html and _layouts/gcode.html which should not be part of the PR.

But after the rebase I still see the changes on the files. Did you indeed changed them ?

**Comment by petrzjunior** [comment]
As I wrote:
I also changed the template for G-Code to diplay author as other pages do. It doesn't break the design and seems helpful to me.

What do you think?

**Comment by jbrazio** [comment]
Silly me I didn't noticed that part.

Then this will be a bit harder to merge because if you see the gcode-info.html and gcode.html have been changed, your edit is overriding some of those changes.

Could you please have only on this PR the M600 document and we will add the author to the html template in a different PR ?

**Comment by petrzjunior** [comment]
Not a problem to split it, but I don't understand:
"if you see the gcode-info.html and gcode.html have been changed"
This PR was made today and nothing new was merged.

**Comment by jbrazio** [comment]
OK let me show you the bigger picture: https://github.com/jbrazio/MarlinDocumentation/commit/51e7cbe4af76b7025fbdd7c23f8de72167515c1b
The layout will change a bit a I prefer to add the author section after this change is merged.

```
$ git rebase -i upstream/master
[ select "edit" instead of "pick" ]
$ git reset --soft HEAD^
$ git reset _includes/gcode-info.html _layouts/gcode.html
$ git checkout _includes/gcode-info.html _layouts/gcode.html
$ git commit --amend --no-edit
$ git push --force
```

PS: We have to keep the "row" and "col-md-12", this is the bootstrap table layout. I know some html templates are not using this [yet] but I've been converting them when I find an errant one.

**Comment by petrzjunior** [comment]
@jbrazio Oh, hidden content... :smile: 
Now I understand. Thank you for your patience. Now it'll be all right I hope.

**Comment by petrzjunior** [comment]
Looks like line
`git reset --soft HEAD^`
isn't working, it is asking `MORE?` What should I write?

**Comment by petrzjunior** [comment]
Looks good now.

**Comment by petrzjunior** [comment]
I will leave it to you and check it in the evening. Hope I've done everything right, you git ninja. :smile:

**Comment by jbrazio** [comment]
It's done ! ;-D

untitled

**Pull request merged**